packages/image_picker/image_picker/test/image_picker_test.dart

```diff
@@ -126,9 +234,51 @@ void main() {
               source: ImageSource.camera,
               preferredCameraDevice: CameraDevice.front);
 
-          verify(mockPlatform.getImage(
-              source: ImageSource.camera,
-              preferredCameraDevice: CameraDevice.front));
+          verify(mockPlatform.getImageFromSource(
+            source: ImageSource.camera,
+            options: argThat(
+              isInstanceOf<ImagePickerOptions>().having(
+                  (ImagePickerOptions options) => options.preferredCameraDevice,
+                  'preferredCameraDevice',
+                  equals(CameraDevice.front)),
+              named: 'options',
+            ),
+          ));
+        });
+
+        test('Full metadata argument defaults to true', () async {
```

Comment:
Minor nit here and below: our usual convention (which the other tests around this follow) is that they start with a lower-case letter.